### ISSUE TITLE
Unnecessary requirements in installation. Remove and publish to conda?

### ISSUE THREAD

**Issue by dkoslicki** [ziyewang/MetaBinner#4]
Taking a look at the `metabinner_env.yaml`, there appears to be a plethora of packages being installed that aren't actually used (appears that the yaml file was generated from a `conda env export -f metabinner_env.yml` or the like, and dumped the whole dev environment). 

For example: `freetype` and `harfbuzz` even though text is neither being manipulated nor added to images, all the `R` packages even though I do not see `R` being used in the code-base, `libtiff` even though images are not being generated, etc. etc.

I imagine only installing what is absolutely necessary (click, biopython, numpy, pandas, mimetypes, scipy, sklearn, biolib, etc.) would make publication on conda/anaconda/bioconda rather straightforward. CheckM itself is on bioconda, so this would further simplify the process.

Let me know if you want any help/pointers in publishing to bioconda!

**Comment by ziyewang** [comment]
Dear dkoslicki,

Thanks very much for your kind reminder. We have generated the yaml file and published the version 1.4 to ensure that our results reported in the manuscript can be fully reproduced. For example, CheckM on bioconda does not support the version we used for python 3.

To make it easier for users to install and make a publication on bioconda, we should indeed release a simplified version. We will try to complete it as soon as possible. Thanks again.

Best,
Ziye

**Comment by dkoslicki** [comment]
Great to hear! With the recent performance of MetaBinner in the [CAMI 2 competition](https://www.biorxiv.org/content/10.1101/2021.07.12.451567v1), I'm sure more people will want to use your tool, so having it on bioconda will ensure they get it working without too much pain.

**Comment by vinisalazar** [comment]
Hi,

@dkoslicki, that would be the case for me. After seeing MetaBinner's performance on CAMI 2, I wanted to try it. However, the lack of a conda recipe (preferably on Bioconda, or elsewhere) has made it difficult to integrate into my existing workflow.

I'm also willing to help if there's interest in a Bioconda recipe. @ziyewang would you welcome a PR to [Bioconda recipes](https://github.com/bioconda/bioconda-recipes) with a draft? I can give it a try, if you agree.

Thanks a lot for providing this package!

Vini

**Comment by ziyewang** [comment]
Hi, vinisalazar,

We are glad that you're interested in this work. I have submitted a PR （https://github.com/bioconda/bioconda-recipes/pull/30048), and it passed the test and wait for review. We have tested it via "conda install -c https://165188-42372094-gh.circle-artifacts.com/0/tmp/artifacts/packages metabinner". Once it passes the review, I will inform you. Thank you very much. Sorry for keeping you waiting. 

Best wishes,
Ziye
